Services/CO.CDP.Organisation.WebApi.Tests/Extensions/ServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,77 @@
+using System.Reflection;
+using Microsoft.AspNetCore.Http;
+using static CO.CDP.Organisation.WebApi.UseCase.RegisterOrganisationUseCase.RegisterOrganisationException;
+using static CO.CDP.OrganisationInformation.Persistence.IOrganisationRepository.OrganisationRepositoryException;
+using static CO.CDP.OrganisationInformation.Persistence.ITenantRepository.TenantRepositoryException;
+
+namespace CO.CDP.Organisation.WebApi.Tests.Extensions;
+
+public class ServiceCollectionExtensionsTests
+{
+    private static (int status, string error) InvokeMapException(Exception exception)
+    {
+        var methodInfo = typeof(CO.CDP.Organisation.WebApi.Extensions.ServiceCollectionExtensions)
+            .GetMethod("MapException", BindingFlags.NonPublic | BindingFlags.Static);
+        return ((int status, string error))methodInfo.Invoke(null, new object[] { exception });
+    }
```

Comment:
There's lots of null related problems reported by static analysis here. 

In general, private methods should be tested through the public interface, not directly. Otherwise, there's no point of them being private and it makes refactoring harder.

Either test this behaviour through the public interface, or make the `MapException` method public please. I think the latter might be actually suitable, as we will need this mapping to later document error responses.